packages/tools/pie-icons-vue/helpers/test/index.test.js

```diff
@@ -0,0 +1,21 @@
+import { removeHyphenBeforeDigits } from '../index';
+
+describe('removeHyphenBeforeDigits', () => {
+    test.each([
```

Comment:
More of a suggestion, It would be nice to split this out so you have just the tests that remove all cases for hyphens and then another test that shows where it won't apply the method i.e `test123` will always be `test123` That way the second test description can read `should NOT remove a hyphen before digits/cases`. At the moment when the test description is read out the test looks like it `should` apply the logic to the cases where it shouldn't remove it if that makes sense.